tox.ini

```diff
@@ -28,6 +28,8 @@ passenv =
   VERIFY_SSL_CERT
   XDG_CACHE_HOME
 
+setenv = HELM_EXPERIMENTAL_OCI = 1
```

Comment:
I'm not sure if this is the best place to set this.

I think we should pass this through as an env-var to common.execute if we expect it to be set all the time in our install.

Reply:
OK I'll see that. I did not know were to place it.